### PR TITLE
added new test plans for BSP, business, businessplus, standard, professional

### PR DESCRIPTION
**Pull request by akarambalkar**
I have created new branch 'zcs-7816' from a develop branch. The earlier branch 'zcs-7731' had lot of merge conflict issues and rebase issues. I tried to fix issues related to rebase , however there were multiple files with multiple line changes and was confusing to fix the same. This new branch I was successfully able to rebase, so I am submitting these test plans for review and PR. Please review and provide feedback for the same.

### CODE REVIEW

**Comment by fatkudu** [comment]
Could the commits be squashed so that the deleting and re-adding of files doesn't leave churn in git history please?

File: build.xml [hunk 1]
```diff
@@ -352,42 +400,6 @@
       <jmeterarg value="-q${basedir}/tests/CSP/pop/profiles/basic.prop"/>
     </jmeter>
   </target>
```
Comment (sakchhi-kiran) [inline build.xml]: seems CSP1 target is missing from build.xml. Could you please check and add the required targe

File: tests/BSP/prec/zsoap/load.prop [hunk 2]
```diff
@@ -0,0 +1,6 @@
+LOAD.duration=600
```
Comment (sakchhi-kiran) [inline tests/BSP/prec/zsoap/load.prop]: do we need to this load profile, as I dont see being called in BSP-prec-zsoap.

File: tests/BSP/prec/zsoap/load_perf.prop [hunk 3]
```diff
@@ -0,0 +1,6 @@
+LOAD.duration=86400
```
Comment (sakchhi-kiran) [inline tests/BSP/prec/zsoap/load_perf.prop]: do we need to this load profile, as I dont see being called in BSP-prec-zsoap.

**Comment by sakchhi-kiran** [comment]
Changes looks good. Verified for the coverage of BSP test as per details provided by Prashant.

File: tests/BSP/pop/profiles/basic.prop [hunk 4]
```diff
@@ -1,9 +1,9 @@
 PROFILE.POP.type=sequence
-PROFILE.POP.sequence.1=XOIP
-PROFILE.POP.sequence.2=LOOP
-PROFILE.POP.sequence.3=USER
-PROFILE.POP.sequence.4=PASS
-PROFILE.POP.sequence.5=NOOP
-PROFILE.POP.sequence.6=RSET
-PROFILE.POP.sequence.7=CAPA
-PROFILE.POP.sequence.8=QUIT
+#PROFILE.POP.sequence.1=XOIP
```
Comment (thefoolwhocodes) [inline tests/BSP/pop/profiles/basic.prop]: @akarambalkar Could you please discuss with @prashantajari about commenting this.

File: bspruntest.sh [hunk 5]
```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
+
+usage=$(
+cat <<EOF
+$0 [OPTION]
+-u VALUE    set "u" argument to specify type of customer
```
Comment (thefoolwhocodes) [inline bspruntest.sh]: Please specify the supported customer types here.

**Comment by thefoolwhocodes** [comment]
@akarambalkar running the following command does not show proper message
ubuntu@ip-10-0-161-97:~/temp/zm-load-testing$ ./bspruntest.sh -u abc
_Customer type not mentioned_

File: tests/BSP/pop/profiles/basic.prop (hunk 4 above)
Comment (akarambalkar) [reply tests/BSP/pop/profiles/basic.prop]: As discussed with @prashantajari, The XOIP request was disabled while running on a beefy machine, However, I have enabled this on OCI environment and executed tests, I have found issues with command, I have discussed with Prashant ajari about the same and will consider to disable this for now, will enabled for future runs when fixed.

File: bspruntest.sh (hunk 5 above)
Comment (akarambalkar) [reply bspruntest.sh]: I have updated the bspruntest.sh as per the review comments, I have removed the previous code and just kept my BSP tests code in this file. This is a separate file and will be used only to execute my BSP specific scenarios.

**Pull request merged**